packages/devextreme/js/__internal/ui/chat/messagegroup.ts

```diff
@@ -83,10 +83,12 @@ class MessageGroup extends Widget<Properties> {
     const { author } = items[0];
     const authorName = author?.name;
     const authorAvatarUrl = author?.avatarUrl;
+    const authorAvatarAlt = author?.avatarAlt;
```

Comment:
I suggest ignoring the ts-error because reviewing the d.ts changes may take some time. Let's make the PR green.
We will remove the comment in the d.ts PR if this one merges first.

Reply:
[2675fac](https://github.com/DevExpress/DevExtreme/pull/28237/commits/2675facd58998531cb47d2ec39ebeffd7ba4a6c9)